app/thread-chat/styles/markdown.css

```
@@ -123,15 +123,15 @@
   overflow: hidden;
-  background: #fff;
+  background: rgba(252,252,252, 1);
 }
 .tc .md-body .md-code-head {
   display: flex;
   align-items: center;
   justify-content: space-between;
   padding: 5px 10px;
-  background: var(--paper-2);
-  border-bottom: 1px solid var(--rule);
+  /*background: var(--paper-2);*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)